src/Xamarin.Android.Build.Tasks/Tasks/GetAdditionalResourcesFromAssemblies.cs

```diff
@@ -307,9 +314,14 @@ string MakeSureLibraryIsInPlace (string destinationBase, string url, string vers
 					Log.LogMessage (MessageImportance.Low, e.ToString ());
 				}
 			}
-			else
-				LogDebugMessage ("    reusing existing archive: {0}", file);
-			string contentDir = Path.Combine (destinationDir, "content");
+			else {
+				if (string.IsNullOrEmpty (extraPath))
+					LogDebugMessage ("    reusing existing archive: {0}", file);
+				else
+					LogDebugMessage ("    found {0} in {1}", embeddedArchive, Path.Combine (AndroidSdkDirectory, extraPath));
```

Comment:
Could the message be more elaborate? What are we looking for?

When logging, we should quote paths, particularly since paths may contain spaces on Windows. I've taken to using backticks.

    LogDebugMessage ("    found SOMETHING archive `{0}` in `{1}`", embeddedArchive, Path.Combine (AndroidSdkDirectory, extraPath));